mosaic-runtime/src/commonMain/kotlin/com/jakewharton/mosaic/mosaic.kt

```diff
@@ -70,12 +70,16 @@ public fun runMosaicBlocking(content: @Composable () -> Unit) {
 }
 
 public suspend fun runMosaic(content: @Composable () -> Unit) {
+	runMosaic(enterRawMode = true, content)
+}
+
+internal suspend fun runMosaic(enterRawMode: Boolean, content: @Composable () -> Unit) {
```

Comment:
I need `awaitComplete` on the tester, but what I really need is to go to bed so punting on that until tomorrow...